### PR TITLE
Update Curator action file information

### PR DESCRIPTION
**Pull request by mburke5678**
Per Josef https://docs.google.com/a/redhat.com/document/d/1-E1z-Z_CvIC7u4ggQFfaH-9aF_p7lIGsdOLD7Zynj9g/edit?disco=AAAAC51-hKQ

### CODE REVIEW

**Comment by mburke5678** [comment]
@josefkarasek Please take a look to see if I have [your comments in the Google doc correct](https://docs.google.com/a/redhat.com/document/d/1-E1z-Z_CvIC7u4ggQFfaH-9aF_p7lIGsdOLD7Zynj9g/edit?disco=AAAAC51-hKQ).

**Comment by mburke5678** [comment]
@anpingli Josef wants to minimize the information around the Curator *action* file and to make sure usrs add an exclude rule to the *action* if they use the file. Please take a look.

File: logging/config/efk-logging-curator.adoc [hunk 1]
```diff
@@ -8,7 +8,18 @@ toc::[]
 The Elasticsearch Curator tool performs scheduled maintenance operations on a per-project basis. Curator performs actions daily based on its configuration. Only one Curator Pod is
 recommended per Elasticsearch cluster.
 
-The cluster logging installation creates a Curator configuration map, `curator` in the `openshift-logging` project, which incorporates the Curator configuration file, *_curator5.yaml_* and an {product-title} custom configuration file, *_config.yaml_*.
+The Cluster Logging Operator installation creates a *Curator* ConfigMap in the `openshift-logging` project, which contains three sections: the link:https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/actionfile.html[Curator *action* file], a `config.yaml` file, and a `curator5` file.  
+
+* *actions*. In {product-title} cluster logging, the Curator `actions` file supports only the *delete_indices* action.
```
Comment (anpingli) [inline logging/config/efk-logging-curator.adoc]: It is better  use "actions.yaml" instead of "actions."

Comment (anpingli) [inline logging/config/efk-logging-curator.adoc]: the Curator `actions` file supports only the *delete_indices* action.
---------
I think it support all actions in  https://www.elastic.co/guide/en/elasticsearch/client/curator/current/actions.html. But only delete_indices are used and tested by openshfit.
@josefkarasek  Could you confirm?

Comment (mburke5678) [reply logging/config/efk-logging-curator.adoc]: @anpingli @josefkarasek I made some updates to the section that Anping commented on (thanks Anping!!). 
```
* *actions.yaml*. The Curator `actions` file.
* *curator5.yaml*. The Curator configuration file for general configuration, such as the Elasticsearch hostname, client connection details, and the path to certificates.
* *config.yaml*. A legacy {product-title} configuration file for Curator. Supports only the Curator *delete_indices* action

```

**Review by anpingli — COMMENTED** [review]
LGTM

**Comment by mburke5678** [comment]
@josefkarasek PTAL 
Preview file:
http://file.rdu.redhat.com/mburke/logging-action-file/logging/config/efk-logging-curator.html

**Review by josefkarasek — CHANGES_REQUESTED** [review]

File: modules/efk-logging-curator-actions.adoc [hunk 2]
```diff
@@ -28,51 +23,40 @@ If you are an advanced Curator user and want to configure other Curator actions,
...
+      timeout_override: '300'
+      ignore_empty_list: true
 ----
-<1> Specify `delete_indices` to delete the specified index.
-<2> Use the `filers` parameters to specify the index to be deleted. See the link:https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/filters.html[Elastic Search curator documentation] for information on these parameters. 
-<3> Specify `false` to allow the index to be deleted.
+<1> Creates `delete_indices` to create the rule.
+<2> Sets the rule to `true` to prevent the index from being deleted.
+<3> Sets the indices to retain. 
+
```
Comment (josefkarasek) [inline modules/efk-logging-curator-actions.adoc]: Item number four - add your custom rules here :)

Comment (mburke5678) [reply modules/efk-logging-curator-actions.adoc]: @josefkarasek Where should the <4> go?

**Comment by mburke5678** [comment]
Closing in favor ot https://github.com/openshift/openshift-docs/pull/14594

**Pull request closed without merging**